lib/src/web_socket_handler.dart

```diff
@@ -66,7 +65,7 @@ class WebSocketHandler {
 
     var protocol = _chooseProtocol(request);
     request.hijack((untypedChannel) {
-      var channel = (untypedChannel as StreamChannel).cast<List<int>>();
+      var channel = untypedChannel.cast<List<int>>();
```

Comment:
If we're no longer casting this, we'll probably need to narrow the constraint on `shelf` to only allow versions where `hijack()` is typed to pass a `StreamChannel`. Also, it shouldn't be called `untypedChannel` anymore, since it is in fact typed.